x-pack/plugins/canvas/canvas_plugin_src/functions/common/joinRows.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ExpressionFunction } from '../../../../../../src/legacy/core_plugins/interpreter/types';
+import { Datatable } from '../types';
+import { getFunctionErrors } from '../../strings/functions';
+
+interface Arguments {
+  column: string;
+  separator: string;
+  quote: string;
+}
+
+const escapeString = (data: string, quotechar: string): string => {
+  if (quotechar === undefined || quotechar === '') {
+    return data;
+  } else {
+    const regex = new RegExp(quotechar, 'g');
+    return data.replace(/\\/g, `\\\\`).replace(regex, `\\${quotechar}`);
+  }
+};
+
+export function joinRows(): ExpressionFunction<'joinRows', Datatable, Arguments, string> {
+  const errors = getFunctionErrors().getCell;
```

Comment:
I don't think the help and error text is meant to be reused like this, this should pull from a new definition for `joinRows`... @clintandrewhall can you confirm that?

Reply:
I would be surprised if two function help or error statements were meant to be identical.  If that's the case, we need to create a `common.ts` dictionary with the shared help/errors:

```
const errors = getFunctionErrors().common;
```

Reply:
I wasn't sure if it was needed. Made a separate error message now.